utils/travis/docker_run.sh

```diff
@@ -176,6 +176,10 @@ else
 
     if [ "$VALIDATE" = "true" ]; then
         execute "WML validation" ./utils/travis/schema_validation.sh
+        cd data/tools/
+        make reindent
+        cd ../../
+        execute "WML indentation check" git diff-index --quiet HEAD
```

Comment:
You could make those commands into a function to have them all called by execute:

```suggestion
        checkindent() {
                make -C data/tools reindent &&
                git diff-index --quiet HEAD
        }
        execute "WML indentation check" checkindent
```

Reply:
Updated.